packages/frontend/amp/lib/tag-utils.ts

```diff
@@ -16,17 +16,23 @@ export type StyledTone =
 
 export const getToneType = (tags: TagType[]): StyledTone => {
     const defaultTone = 'default-tone';
-    const tone = filterForTagsOfType(tags, 'Tone')[0];
+    const tones = filterForTagsOfType(tags, 'Tone').map(tone => tone.id);
 
-    if (!tone) {
+    if (!tones) {
         return defaultTone;
     }
 
-    switch (tone.id) {
+    // Editorials have two tones - tone/editorials and tone/comment
+    // We defer to tone/comment for styling
+    if (tones.includes('tone/editorials')) {
+        return 'tone/comment';
+    }
+
+    switch (tones[0]) {
```

Comment:
I'm wondering if this function can be written a bit more clearly. Is the tone/editorial always first if present (or if not, then the tone/comment one is?). If so, we could just add it to the switch? E.g.

    switch (tone) {
        case 'tone/editorials':
            return 'tone/comment';
        ..
    }

Reply:
Yes, I had it as a fourth switch case initially and will revert back to this. It should catch either tone/comment or tone/editorial. In the articles I spot checked editorial was always first.